scripts/scripts.js

```diff
@@ -394,6 +394,32 @@ export function decorateMain(main) {
   addTopSpacingStyleToFirstMatchingSection(main);
 }
 
+async function loadTemplate(doc, templateName) {
```

Comment:
`decoratePageStyles` does something similar already.  Why do we need `loadTemplate` ?

Reply:
@sdmcraft  - This is a bit different AFAIK, from the Block Party - any kind of page templates added, that template should have its own corresponding js & css file - [Dynamically loaded Templates](https://github.com/hlxsites/blogs-keysight/blob/fbf6bba5989bd25f699bf2c78737e4c820bdd3e4/blogs/scripts/scripts.js#L393-L417)

https://github.com/hlxsites/blogs-keysight/blob/fbf6bba5989bd25f699bf2c78737e4c820bdd3e4/blogs/scripts/scripts.js#L393-L417

Thats why added.

Reply:
ok, it looks like yet another way of doing what `decoratePageStyles` already does. If you'd want to keep this, they please remove `decoratePageStyles` . We need only one of these.

Reply:
@sdmcraft  - I was not aware about decoratePageStyles which makes more sense here since here we just need styling in css, nothing in js. With loadtemplate, there is a mandate to add a .js file which is not the case with decoratePageStyles. I have modified the document metadata to leverage page-style rather instead of template. 

https://issue384-underfill-page--sunstar--hlxsites.hlx.live/_drafts/meet/underfill

is working with page-style metadata now. Pl check and validate. 

Many Thanks.